### ISSUE TITLE
Indicate fallback language to the browser when used from Model.Value

### ISSUE THREAD

**Issue by RachBreeze** [umbraco/Umbraco-CMS#9321]
Hello 

For accessibility purposes, it's important that the correct language is defined on the page see https://www.youtube.com/watch?v=QwOoU8T24UY&feature=youtu.be for a demo.

When working on multilingual sites it's possible to define a fallback language, so if a page doesn't have content in one language, content is retrieved in another.  https://our.umbraco.com/Documentation/Getting-Started/Design/Rendering-Content/#using-fall-back-methods documents how to use fallback methods.  

However when a fall back is used the mark up should indicate this is the case 
https://www.w3.org/International/questions/qa-html-language-declarations#:~:text=Quick%20answer,an%20element%20surrounding%20that%20content. 

I _think_ the solution is to use a style similar `@model.ValueFor` and wrap the value returned in a `span`, when a fallback language is used.  I would love to help resolve the issue but recognise there are some things that need bottoming out first!

**Comment by nul800sebastiaan** [comment]
@RachBreeze  We don't want to mess with the HTML output as it is, we're in fact pretty proud that Umbraco usually only outputs exactly what you ask for. I would suggest that we can add an additional optional parameter to `Model.Value` that will surround the output with something like `<span lang="zh-Hans">`, but only if a fallback language is picked of course. 

Make sense?

**Comment by umbrabot** [comment]
Hi @RachBreeze,

We're writing to let you know that we would love some help with this issue. We feel that this issue is ideal to flag for a community member to work on it. Once flagged here, folk looking for issues to work on will know to look at yours. Of course, please feel free work on this yourself ;-).  If there are any changes to this status, we'll be sure to let you know. 

For more information about issues and states, have a look at [this blog post](https://umbraco.com/blog/the-umbraco-issue-tracker-process/)

Thanks muchly, from your friendly Umbraco GitHub bot :-)

**Comment by RachBreeze** [comment]
PR #9576 has been created to resolve this issue
